ecommerce/extensions/api/v2/views/coupons.py

```diff
@@ -408,8 +408,8 @@ def update_range_data(self, request_data, vouchers):
         # In case of enterprise, range_data has enterprise data in it as enterprise is defined in UPDATABLE_RANGE_FIELDS
         # so Catalog should not be None if there is an enterprise is associated with it.
         if voucher_range.catalog:
-            if not enterprise_customer_data:
-                range_data['catalog'] = None
+            # if not enterprise_customer_data:
+            #     range_data['catalog'] = None
```

Comment:
Can you add a comment explaining a bit what's happening?

Reply:
yes please, I think it's necessary to leave a comment explaining why we are commenting these lines

Reply:
Ready, if you have any suggestions, I'm attentive.
Thanks @mariajgrimaldi @magajh

Reply:
Change the wording

The following two lines set voucher_range.catalog_id to "None", losing the course_id associated with the coupon.